pgjdbc/src/test/java/org/postgresql/test/jdbc2/DriverTest.java

```diff
@@ -275,6 +277,40 @@ public void testSetLogStream() throws Exception {
     }
   }
 
+  @Test
+  public void testSystemErrIsNotClosedWhenCreatedMultipleConnections() throws Exception {
+    TestUtil.initDriver();
+    PrintStream err = System.err;
+    String loggerLevel = System.getProperty("loggerLevel");
+    String loggerFile = System.getProperty("loggerFile");
+
+    System.clearProperty("loggerLevel");
+    System.clearProperty("loggerFile");
+    System.setProperty("loggerLevel", "INFO");
+    PrintStream buffer = new PrintStream(new ByteArrayOutputStream());
+    System.setErr(buffer);
+    try {
+      Connection con = DriverManager.getConnection(TestUtil.getURL());
```

Comment:
This might need  the user and password: 
`DriverManager.getConnection(TestUtil.getURL(), TestUtil.getUser(), TestUtil.getPassword());`